portfolio/src/main/webapp/script.js

```diff
@@ -42,10 +42,52 @@ function fadeOut(){
 }
 
 //From Week-3 Tutorial
-function getGreeting() {
-  console.log('Handling the response.');
-  fetch('/data').then(response => response.text()).then((quote) => {
-    document.getElementsByClassName('greeting-cont')[0].innerText = quote;
+function getComments() {
+  //Get the number of comments to show per page
+  numComments = localStorage.getItem("num-of-comments");
+  fetch(`/comments?num-comments=${numComments}`).then(response => response.json()).then((comment) => {
+    container = document.getElementsByClassName('comments-section')[0];
+    container.innerHTML = "";
+    for (item in comment) {
```

Comment:
`for (const comment of comments) {`

also rename comment -> comments since comment is really an array of comment objects right

Reply:
done